README.md

```diff
@@ -1 +1,71 @@
+## Bougie Brews
 
+Bougie Brews is a fullstack e-commerce site selling beer. Bougie Brews represents the Grace Shopper project in senior phase at Fullstack Academy.
```

Comment:
Thanks for adding a Readme! I would recommend replacing “beer” with “microbrews” or “craft beer”. Makes things sound more classy